pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -169,14 +169,16 @@ func (ks *scaler) handleScaleToZero(pa *pav1alpha1.PodAutoscaler, sks *nv1a1.Ser
 
 		// Do not scale to 0, but return desiredScale of 0 to mark PA inactive.
 		sw := aresources.StableWindow(pa, config)
-		if pa.Status.CanMarkInactive(sw) {
+		af := pa.Status.ActiveFor()
+		if af >= sw {
 			// We do not need to enqueue PA here, since this will
 			// make SKS reconcile and when it's done, PA will be reconciled again.
 			return desiredScale, false
 		}
-		// Otherwise, scale down to 1 until the idle period elapses and re-enqueue
-		// the PA for reconciliation at that time.
-		ks.enqueueCB(pa, sw)
+		// Otherwise, scale down to at most 1 for the remainder of the idle period and then
```

Comment:
„At least 1“?

Reply:
Eh, since this is scale to 0, so "at most 1", as in no more than 1, but still not 0...
Perhaps just "scale to 1".